redisson/src/main/java/org/redisson/spring/session/RedissonSessionRepository.java

```diff
@@ -214,14 +214,23 @@ public String changeSessionId() {
             Long remainTTL = (Long) list.get(0);
             Map<String, Object> oldState = (Map<String, Object>) list.get(2);
 
-            RBatch batchNew = redisson.createBatch(BatchOptions.defaults());
```

Comment:
Please use RBatch object

Reply:
@mrniko thanks, reverted to batch. While testing the fix, I thought we were getting exceptions when a batch only had a single command, but I seem to have been confused.